var/spack/repos/builtin/packages/py-dask/package.py

```diff
@@ -130,7 +136,7 @@ class PyDask(PythonPackage):
     depends_on('py-toolz@0.8.2:',       type=('build', 'run'), when='@2.13.0: +delayed')
 
     # Support for YAML configuration files
-    depends_on('py-pyyaml',             type=('build', 'run'), when='+yaml')
+    depends_on('py-pyyaml',             type=('build', 'run'))
```

Comment:
@a-saitoh-fj Shouldn't this be `when='+yaml'`? Otherwise, the `yaml` variant does nothing.